pom.xml

```diff
@@ -4,7 +4,7 @@
     <modelVersion>4.0.0</modelVersion>
```

Comment:
Doesn't look like the pom actually enabled DC clustered testing.  How are you validating that?

https://github.com/Libertymutual/ssh-key-enforcer-stash/blob/master/pom.xml#L228 whos how you can define multiple instances to start via atlas-run, atlas-verify using a shared DB to make sure DataCenter is actually supported, and not triggering duplicate events, etc.

https://github.com/Libertymutual/ssh-key-enforcer-stash/blob/master/runCluster.sh has some commands that cleaned up directories on each run and reduced expected DB pool to fit Cloud Foundry's MySQL connection limit, may not be needed any longer.